src/applications/personalization/profile/components/personal-information/ContactInformationEditView.jsx

```diff
@@ -73,16 +73,30 @@ export class ContactInformationEditView extends Component {
     validateAddress: PropTypes.func.isRequired,
   };
 
+  focusOnFirstFormElement() {
```

Comment:
the logic in here is very similar to the logic that's used (or used to be used) in the Modal component

Reply:
I can't believe I haven't seen `querySelector` with comma separated values before.

Reply:
YOU'RE welcome!